### DISCUSSION TITLE
Pacific mod file

### DISCUSSION THREAD

**Discussion by crusher42** [gorilla-devs/ferium#354] [Q&A]
hi all
i need a Pacific version of a mod is there any way of adding that

what I mean it need to stay at that version unless if I change it myself.

for example
fabric API & optifabric "mc 1.19.4" don't work after fabric API  updated past version 0.79.0 

i try doing this

`
        {
          "name": "Fabric API",
          "identifier": {
            "CurseForgeProject": 306612::4503572
          }
        },`

**Comment by theRookieCoder** [comment]
This feature comes under #82, currently you'll have to manually download the mod

**Reply by crusher42** [reply]
oh thx for the info
i will be waiting to see the update

i know coding takes time
